features/reprint_summary_document.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Reprint Summary Document
+  As a user
+  I want to reprint a customers summary document
+  So that I can resend it when the customer has not received the original
+
+  @unauthenticated
+  Scenario: Reprint a summary document
+    Given I log in to reprint a summary document
+    When I enter a customer reference number
```

Comment:
Is it a customer reference number? Or an appointment reference? How do they refer to it?